test/integration/objectid.test.ts

```diff
@@ -0,0 +1,142 @@
+import { expect } from 'chai';
+
+import { Collection, Db, MongoClient, ObjectId } from '../../src';
+import { sleep } from '../tools/utils';
+
+describe('ObjectId', function () {
```

Comment:
this entire test file is kind of a trainwreck with a mix of unit tests and setup/cleanup not handled appropriately - I understand that it's out of scope for this work, but please file a technical debt follow up to fix this specific test file

Reply:
https://jira.mongodb.org/browse/NODE-4989
and left a todo